server/jest.config.ci.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Internal dependencies
+ */
+const jestConfig = require( './jest.config' );
+
+module.exports = Object.assign(
+	{},
+	{
+		testResultsProcessor: './node_modules/jest-junit-reporter',
```

Comment:
It was quite easy to integrate a reporter that is going to play nicely with Circle CI 🎉